src/Raven.Studio/typescript/models/resources/widgets/databaseNotificationsItem.ts

```diff
@@ -0,0 +1,83 @@
+class databaseNotificationsItem implements databaseAndNodeAwareStats {
+    database: string;
+    nodeTag: string;
+    
+    alertsCount: number;
+    alerts: Raven.Server.Dashboard.Cluster.Notifications.DatabaseNotifications.NotificationSummaryItem[];
+
+    performanceHintsCount: number;
+    performanceHints: Raven.Server.Dashboard.Cluster.Notifications.DatabaseNotifications.NotificationSummaryItem[];
+
+    noData: boolean;
+
+    hideDatabaseName: boolean;
+    even = false;
+
+    constructor(nodeTag: string, data: Raven.Server.Dashboard.Cluster.Notifications.DatabaseNotifications.DatabaseNotificationsSummary) {
+        this.nodeTag = nodeTag;
```

Comment:
```suggestion
    constructor(private nodeTag: string, data: Raven.Server.Dashboard.Cluster.Notifications.DatabaseNotifications.DatabaseNotificationsSummary) {
```

and remove `nodeTag: string` at 3 line

Reply:
It needs to be this way because this class `implements databaseAndNodeAwareStats`